web-ui/app/js/mail_view/ui/draft_box.js

```diff
@@ -72,6 +72,26 @@ define(
         this.enableAutoSave();
         this.on(this.select('cancelButton'), 'click', this.showNoMessageSelected);
         this.on(this.select('closeMailButton'), 'click', this.showNoMessageSelected);
+
+        function floatlabelHandler(element) {
```

Comment:
same function declared twice is not very nice :(
would it be possible to use the [mixins feature](https://github.com/flightjs/flight/blob/master/doc/mixin_api.md) from flightjs?